src/pages/library/DataBrowser.js

```diff
@@ -221,11 +225,30 @@ const Browser = () => {
   const [selectedData, setSelectedData] = useState([])
   const [requestDatasetAccessList, setRequestDatasetAccessList] = useState()
   const { dataCatalog, loading } = useDataCatalog()
+  const acknowledged = Utils.useStore(acknowledgmentStore)
 
   const toggleSelectedData = data => setSelectedData(_.xor([data]))
 
   return h(FooterWrapper, { alwaysShow: true }, [
     libraryTopMatter('browse & explore'),
+    !acknowledged && div({
+      style: {
+        border: `1px solid ${colors.accent()}`, borderRadius: 3,
+        backgroundColor: 'rgba(0,0,0,.1)',
+        padding: '5px 20px', margin: 20,
+        fontSize: '.8rem', fontWeight: 'bold',
+        display: 'flex', flexDirection: 'row', alignItems: 'center', justifyContent: 'space-between'
+      }
+    }, [
+      div({ style: { lineHeight: '1.4rem' } }, [
+        div(['Thank you for participating in the BETA Catalog test. The HCA data is for the sole use of the BETA testing of the Terra Data Catalog.']),
+        div(['Not for research purposes.'])
+      ]),
+      h(ButtonPrimary, {
+        style: { minWidth: 88, minHeight: 40 },
+        onClick: () => acknowledgmentStore.set(true)
```

Comment:
Set the pref with the ID
```suggestion
        onClick: () => acknowledgmentStore.set({ [id]: true })
```